src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs

```diff
@@ -436,6 +437,33 @@ private async Task StopAsyncCore(bool disposing)
             }
         }
 
+#if NETCOREAPP3_0
+
+        /// <summary>
+        /// Invokes a streaming hub method on the server using the specified method name, return type and arguments.
+        /// </summary>
+        /// <typeparam name="TResult">The return type of the streaming server method.</typeparam>
+        /// <param name="methodName">The name of the server method to invoke.</param>
+        /// <param name="args">The arguments used to invoke the server method.</param>
+        /// <param name="cancellationToken">The token to monitor for cancellation requests. The default value is <see cref="CancellationToken.None" />.</param>
+        /// <returns>
+        /// A <see cref="IAsyncEnumerable{TResult}"/> that represents the stream.
+        /// </returns>
+        public async Task<IAsyncEnumerable<TResult>> StreamAsyncCore<TResult>(string methodName, object[] args, CancellationToken cancellationToken = default)
+        {
+            var stream = CastIAsyncEnumerable<TResult>((await StreamAsChannelCoreAsync(methodName, typeof(TResult), args, cancellationToken)).ReadAllAsync());
```

Comment:
Have to wrap the result of the `ReadAllAsync()` in a cast helper method because all of our streaming methods use `Channel<object>` which comes out to IAsyncEnumerable<object>